sdk/src/test/java/io/dapr/actors/runtime/ActorTypeInformationTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for ActorTypeInformation.
+ */
+public class ActorTypeInformationTest {
+
+    /**
+     * Actor interfaced used in this test only.
+     */
+    private interface MyActor extends Actor {
+    }
+
+    /**
+     * Checks information for a non-remindable actor.
+     */
+    @Test
+    public void notRemindable() {
+
+        class A extends AbstractActor implements MyActor {
+        }
+
+        ActorTypeInformation info = ActorTypeInformation.parse(A.class);
+        Assert.assertNotNull(info);
+        Assert.assertEquals("A", info.getName());
+        Assert.assertEquals(A.class, info.getImplementationClass());
+        Assert.assertFalse(info.isAbstractClass());
+        Assert.assertFalse(info.isRemindable());
+        Assert.assertEquals(1, info.getInterfaces().size());
+        Assert.assertTrue(info.getInterfaces().contains(MyActor.class));

```

Comment:
@amanbha In this example, should it return only MyActor (like it is now) or also Actor. Is the expected result is a shallow list of interfaces, then it is correct. Otherwise, we should do a deep search. This is the same behavior I identified in the .Net SDK as well.